src/Storage/Storage.Management/StorageAccount/GetAzureStorageEncryptionScope.cs

```diff
@@ -68,6 +68,21 @@ public class GetAzureStorageEncryptionScopeCommand : StorageFileBaseCmdlet
         [ValidateNotNullOrEmpty]
         public string EncryptionScopeName { get; set; }
 
+        [Parameter(Mandatory = false,
+            HelpMessage = "The maximum number of encryption scopes that will be included in the list response")]
+        public int? MaxPageSize { get; set; }
+
+        [Parameter(Mandatory = false,
+            HelpMessage = "The filter of encryption scope name. When specified, only encryption scope names starting with the filter will be listed. eg: startswith(name, <prefix>)")]
```

Comment:
Might better be:
"The filter of encryption scope name. When specified, only encryption scope names starting with the prefix in the filter will be listed. eg: startswith(name, <prefix>)"

Reply:
Done